posthog/api/exports.py

```diff
@@ -32,11 +37,17 @@
 
 
 def get_content_response(asset: ExportedAsset, download: bool = False):
-    res = HttpResponse(asset.content, content_type=asset.export_format)
+    content = asset.content
+    if not content and asset.content_location:
+        content_bytes = object_storage.read_bytes(asset.content_location)
+        content = gzip.decompress(content_bytes)
```

Comment:
Random thought but here we decompress the content with gzip, then create a httpresponse which will (maybe?) gzip it again. I don't know Django well enough to say but can we create a `GzippedHttpResponse` or something to skip the unnecessary decompression?

Reply:
Yeah this is what I'm not sure about either. Perhaps it may make sense to leave the creation of the export in the relevant api endpoint but use the GET functions of the exports endpoint for the actual retrieval? Food for thought...